Shared/ExampleObservableRoom.swift

```diff
@@ -18,8 +18,27 @@ extension ObservableParticipant {
     }
 }
 
+struct RoomMessage: Identifiable, Equatable, Hashable, Codable {
+    // message id
+    let id: String
```

Comment:
Looks like this is a message id. Can you please rename this to massageId so that it us clear?